virttest/utils_misc.py

```diff
@@ -2823,13 +2830,15 @@ def format_windows_disk(session, did, mountpoint=None, size=None,
             format_cmd = ' '.join([cmd_header, format_cmd, cmd_footer])
             session.cmd(format_cmd, timeout=300)
 
-            return True
+            return mountpoint
```

Comment:
There is no code to seize the mountpoint when using the default mountpoint=None, so it would return None.